src/modals/components/CreateProject.svelte

```diff
@@ -82,7 +82,7 @@
       return "";
     }
 
-    if (name === "") {
+    if (name.match(/^\s*$/)) {
```

Comment:
```suggestion
    if (name.trim() === "") {
```

To avoid regular expressions, I think this would work the same, right?

Reply:
Quite sure! As we can only type space char, the RegEx is indeed unnecessary. 
Follow-up commits are just submitted. I will also recall the RegExes used in filenames and use trim instead. (Refer to PR #421)